api/src/main/resources/workbench.yaml

```diff
@@ -4324,3 +4327,76 @@ definitions:
         type: "array"
         items:
           $ref: "#/definitions/Institution"
+
+  OrganizationType:
+    type: string
+    description: A categorization of institutions, derived from CAPS requirements
+    enum: [ACADEMIC_RESEARCH_INSTITUTION, INDUSTRY, EDUCATIONAL_INSTITUTION, HEALTH_CENTER_NON_PROFIT, OTHER]
+
+  InstitutionalRole:
+    type: string
+    description: >
+      The union of the roles researchers can have at Institutions of all OrganizationTypes,
+      derived from CAPS requirements.
+      InstitutionalRole_ARI, InstitutionalRole_IND_HCNP, and InstitutionalRole_EDU are
+      subsets of this enum, implemented using the same short-int values.
+    enum: [
+      UNDERGRADUATE, TRAINEE, FELLOW, EARLY_CAREER, MID_CAREER, LATE_CAREER,
+      PRE_DOCTORAL, POST_DOCTORAL, PI,
+      TEACHER, STUDENT, ADMIN,
+      PROJECT_PERSONNEL, OTHER
+    ]
+
+  InstitutionalRole_ARI:
```

Comment:
These 3 are strictly for the UI.  They might fit better in the UI code as typescript enums.